src/modules/list.rb

```diff
@@ -5,7 +5,7 @@ def load_books
     else
       puts @books.count <= 1 ? "\n#{@books.count} Book\n" : "\n#{@books.count} Books \n"
       @books.each_with_index do |book, index|
-        puts "#{index + 1} Title: '#{book.title}', Author: #{book.author}"
+        puts "#{index + 1} Title: '#{book['title']}', Author: #{book['author']}"
       end
```

Comment:
- [ ] When listing a rental, I noticed 3 issues and it would be nice if you resolved them so that your Application works the same way as shown in the video:

1. I get the error below when I try to list the rental

![image](https://user-images.githubusercontent.com/77106962/225159296-40b3e319-e260-4af7-b6c7-c56215af76cc.png)

The issue is that there is no local variable named `rental` instead we have an instance variable called `@rental`. So I would advise you to loop through the instance variable. 💻 👍 

2. The way you are accessing individual items in the `@rentals` array is wrong. Since each item is a Hash, you need to use the bracket notation to access them. For Example, `rent['date']`. Also, you need to crosscheck the way you are storing the information in the _**rentals.json**_ file. There are no keys for people and books in the items, so no need to do `people.name` or `book.title.` You just need to access the `name` key and the `title` key. Kindly implement this. 😎 💻 
3. Based on the video provided as a template, the way you are listing the _**rentals**_ is incorrect. You are currently listing all the _**rentals**_ stored, whereas, you ought to list all _**rentals**_ for a particular person using the `ID` attribute of the `person`. I would advise you to adjust your logic so that your application works the same way as shown in the video. 👍 😎